deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/JobPropertiesParser.java

```diff
@@ -270,7 +296,7 @@ public int getProgress()
      *
      * @return The value of excludeKeysInExport
      */
-    public boolean isExcludeKeysInExport()
+    public Boolean isExcludeKeysInExport()
     {
```

Comment:
As this is a public API I would be hesistant to make this change. If we absolutely have to then we should follow the deprecation process here. Can you justify why?

Reply:
I see, is there a better way to do this? -> https://github.com/Azure/azure-iot-sdk-java/pull/717/files#diff-85f710f651a02e7437edb830cfaafa4fR737

This is the main reason I had to change it to Boolean for null checking using JobProperties.

Reply:
So if it is boolean and not Boolean then customers can never set it to null and will be forced to provide a value. So I don't see value here in this check. What was the motivation for this check?

Reply:
That's only true for earlier APIs. In new API it is JobProperties and if excludeKeys is not set, we are sending the request to service without mandatory requirements. Motivation of this check is to make sure these two necessary parameters are provided at least.

Reply:
As per our conversation offline, removing the validation for import export API and reverting back changes for boolean -> Boolean